schemas/user.py

```
@@ -0,0 +1,167 @@
+# 이제 클라이언트가 회원가입할때
+class CreateUser(BaseModel):
+    email: EmailStr
+    password: str = Field(..., min_length=8, description="8자 이상, 영문/숫자/특수 문자 포함")
+    nickname: str
+    profile_image: str | None = None
+
+    @field_validator('password')
+    @classmethod
+    def validate_password(cls, v: str) -> str:
+        if len(v) < 8:  # 최소 8자 이상
+            raise ValueError('비밀번호는 최소 8자 이상이어야 합니다')
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)